### DISCUSSION TITLE
1.0.4 BETA

### DISCUSSION THREAD

**Discussion by k-ool** [Manafia-Development/Manafia-Factions#5] [General]
**NEW BETA RELEASE**

New Features:

- Lunar Client Integration
- Faction Raids 
- Faction Cloaks
- More upgrades
- Bug Fixes
- Started to add more HCF features

Note: This is a beta. All of these new features are just basic implementations and we are nowhere near done. 

Please report bugs!

<hr /><em>This discussion was created from the release <a href='https://github.com/Manafia-Development/Manafia-Factions/releases/tag/v1.0.4-BETA'>1.0.4 BETA</a>.</em>

**Comment by ShzPlayz** [comment]
Very nice!
